src/main/java/org/springframework/integration/aws/inbound/kinesis/KinesisMessageDrivenChannelAdapter.java

```diff
@@ -339,6 +343,17 @@ public void setBindSourceRecord(boolean bindSourceRecord) {
 		this.bindSourceRecord = bindSourceRecord;
 	}
 
+	/**
+	 * Specify a {@link Function Function&lt;List&lt;Shard&gt;, List&lt;Shard&gt;&gt;} to map the detected shards to
+	 * consume with user business logic.
+	 *
```

Comment:
No blank lines in the method JavaDocs, please: https://github.com/spring-projects/spring-framework/wiki/Code-Style#javadoc-formatting